core/osrd-geom/src/main/java/fr/sncf/osrd/geom/Point.java

```diff
@@ -10,15 +10,17 @@ public record Point(
 
     /**
      * Returns the distance between this point and another in meters, assuming x = longitude and y =
-     * latitude
+     * latitude. Uses equirectangular distance approximation (very fast but not 100% accurate)
      */
     public double distanceAsMeters(Point other) {
-        GeodeticCalculator geoCalc = new GeodeticCalculator();
-        Ellipsoid reference = Ellipsoid.WGS84;
-        GlobalPosition thisPosition = new GlobalPosition(y, x, 0.0);
-        GlobalPosition otherPosition = new GlobalPosition(other.y, other.x, 0.0);
-        return geoCalc.calculateGeodeticCurve(reference, thisPosition, otherPosition)
-                .getEllipsoidalDistance();
+        var earthRadius = 6_378_160;
```

Comment:
I feel like this should set off a `warning: magic number`. Put it into a final static value?

Reply:
We don't really have a magic number warning

I can make it final but not static without moving it out of the method (which would be weird, it's an "internal" aspect). I don't think this is a wrong use of magic number anyway, we do need the info and I don't expect it to change soon